{"name": "test_create_new_pr", "status": "failed", "statusDetails": {"message": "AssertionError: assert 401 == 201\n +  where 401 = <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>>()\n +    where <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>> = <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>.get_status_code", "trace": "def test_create_new_pr():\n        response = api.send_post(pulls, auth=auth, json=params)\n        pr_create_response = Apiresponses(response)\n>       assert pr_create_response.get_status_code() == 201\nE       assert 401 == 201\nE        +  where 401 = <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>>()\nE        +    where <bound method Apiresponses.get_status_code of <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>> = <api.api_responses.Apiresponses object at 0x000001B77FB3DB70>.get_status_code\n\ntest_all_together.py:65: AssertionError"}, "start": 1679227576629, "stop": 1679227576886, "uuid": "6553f57c-157d-4f04-b152-e50830512847", "historyId": "41b560b1a59af6565535d677d61eb1b3", "testCaseId": "0b820b48968caf55db6ab716d7810690", "fullName": "test_all_together#test_create_new_pr", "labels": [{"name": "suite", "value": "test_all_together"}, {"name": "host", "value": "AlphaTest"}, {"name": "thread", "value": "2344-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_all_together"}]}